src/components/EditableCard.tsx

```diff
@@ -62,26 +67,40 @@ class EditableCard extends Component<IEditableCardProps> {
   }
 
   private get deleteButton() {
-    const { isGuarded, classNameSuffix, onDelete, isLoading, title } = this.props,
-      className = getBtnClassName('delete', classNameSuffix, title);
+    const {
+        isGuarded,
+        classNameSuffix,
+        deleteDisabledTooltip,
+        isLoading,
+        model,
+        onDelete,
+        protectedField,
+        title,
+      } = this.props,
+      className = getBtnClassName('delete', classNameSuffix, title),
+      hasProtectedObject = protectedField ? !!model && model[protectedField].length : false;
 
     if (!onDelete) {
       return null;
     }
 
     return (
-      <GuardedButton
-        className={className}
-        confirm={true}
-        disabled={isLoading || this.isDeleting.isTrue}
-        icon={<DeleteOutlined />}
-        isGuarded={isGuarded}
-        onClick={this.handleDelete}
-        size="small"
-        type="danger"
-      >
-        Delete
-      </GuardedButton>
+      <Tooltip title={hasProtectedObject ? deleteDisabledTooltip : ''}>
+        <span>
```

Comment:
Wrapping it in a `<span>` is necessary because tooltips don't show on disabled buttons